SingularityUI/app/components/common/formItems/Duration.jsx

```diff
@@ -29,7 +29,7 @@ export default class Duration extends React.Component {
             min={0}
             step={1}
             value={this.props.value ? duration.hours() : ''}
-            onChange={(e) => this.handleChange(e)}
+            onChange={(event) => this.handleChange(event)}
```

Comment:
When making this change, I found a bug (that is present in `decaf` too). I'm working on fixing that.